### PR TITLE
kb(grid): Revamp Custom Delete Confirmation examples

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: knowledge-base/grid-customize-delete-confirmation-dialog.md [hunk 1]
```diff
@@ -6,18 +6,19 @@ page_title: Customize the Delete Confirmation Dialogs
...
-	<tbody>
-		<tr>
-			<td>Product</td>
-			<td>Grid for Blazor <br/> TreeList for Blazor <br/> Scheduler for Blazor </td>
-		</tr>	
-	</tbody>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Grid for Blazor <br/> TreeList for Blazor <br/> Scheduler for Blazor </td>
```
Comment (ikoevska) [inline knowledge-base/grid-customize-delete-confirmation-dialog.md:19]:
```suggestion
            <td>Grid for Blazor <br/>TreeList for Blazor <br/>Scheduler for Blazor </td>
```

File: knowledge-base/grid-customize-delete-confirmation-dialog.md [hunk 2]
```diff
@@ -6,18 +6,19 @@ page_title: Customize the Delete Confirmation Dialogs
 slug: grid-kb-customize-delete-confirmation-dialog
 position: 
 tags: grid, customize, delete, confirmation, dialog, message
-ticketid: 1553006
+ticketid: 1553006, 1660439, 1625888
 res_type: kb
 ---
 
 ## Environment
```
Comment (yordan-mitev) [inline knowledge-base/grid-customize-delete-confirmation-dialog.md:13]: Consider editing the Description section by applying the current standards with "This KB also answers..."

File: knowledge-base/grid-customize-delete-confirmation-dialog.md [hunk 3]
```diff
@@ -63,70 +64,120 @@ The keys for the elements of the built-in Delete Confirmation Dialog are:
 
 Use [Predefined Confirm Dialog]({%slug dialog-predefined%}#confirm) with the desired custom text. Additionally, you may get the details for the current item and add them to the text:
```
Comment (yordan-mitev) [inline knowledge-base/grid-customize-delete-confirmation-dialog.md:65]:
```suggestion
Use a [Predefined Confirm Dialog]({%slug dialog-predefined%}#confirm) with the desired custom text. Additionally, you may get the details for the current item and add them to the text:
```

**Review by yordan-mitev — APPROVED** [review]

File: knowledge-base/grid-customize-delete-confirmation-dialog.md [hunk 4]
```diff
@@ -63,70 +64,120 @@ The keys for the elements of the built-in Delete Confirmation Dialog are:
...
 
-* Handle the [`OnDelete`]({%slug grid-events%}#cud-events) event of the Grid
-* Display the Predefined Dialog in the `OnDelete` handler
-* Cancel the event or proceed with the `OnDelete` logic depending on the user choice
+* Use the Grid `Class` parameter to set a `z-index` style, which is lower than the default Dialog `z-index` of 10,000.
+* Handle the [`OnDelete`]({%slug grid-events%}#cud-events) event of the Grid.
+* Display the predefined Dialog in the `OnDelete` handler.
+* Cancel the event or proceed with the `OnDelete` logic depending on the user choice.
+* The same approach is applicable for the `OnCreate` and `OnUpdate` events.
```
Comment (ikoevska) [inline knowledge-base/grid-customize-delete-confirmation-dialog.md:71]: I recommend going with `applicable to`. [Source 1](https://twominenglish.com/applicable-to-or-for/) and [source 2](https://textranch.com/c/applicable-for-or-applicable-to/).

**Review by ikoevska — APPROVED** [review]

**Pull request merged**